static/src/javascripts/projects/common/modules/commercial/krux.js

```diff
@@ -92,7 +92,7 @@ export const getKruxSegments = (): Array<string> => {
 };
 
 export const krux: ThirdPartyTag = {
-    shouldRun: config.switches.krux,
+    shouldRun: !!config.get('switches.krux'),
```

Comment:
Minor: To avoid the double negation, could we use the fallback value instead:

```js
shouldRun: config.get('switches.krux', false),
```
or does Flow complain?

Reply:
`flow` may complain as the return type of `config.get` is `any`, I'll try using the fallback to see how `flow` behaves and if it's OK then I'll make this change.

Reply:
`config.get()` should probably return [mixed](https://flow.org/en/docs/types/mixed/#toc-anything-goes-in-nothing-comes-out) and to allow refinements. That's definitely another story, though!